ToDoList/module/service/NetworkManager.swift

```diff
@@ -25,7 +25,15 @@ class NetworkManager {
     
     private init() {}
     
-    func fetch<T: Decodable>(url: URL, decodeType: T.Type) async -> Result<T, NetworkError> {
+    func request<T: Decodable>(url: URL, method: HTTPMethod = .GET, body: Data? = nil, decodeType: T.Type) async -> Result<T, NetworkError> {
```

Comment:
it would be better to create enpoint structure to define  baseUrl, path, method, body and headers

Reply:
Created endpoint structure to define base URL, path, method, body and headers